week-1-ts-ecosystem/w01e03-object-oriented-structure/src/modules/cart/cart.module.ts

```diff
@@ -0,0 +1,64 @@
+import { Product, ProductType } from "../product/product.module";
+
+type currency = "PLN" | "USD" | "EUR";
+export class ShoppingCart {
+  #products: Product[] = [];
+  #name: string;
+  #currency: currency;
+  #total: number = 0;
```

Comment:
taki zapis: 
```
#total: number = 0;
```
Nie jest konieczny pod względem typu.
Wystarczy:
```
#total = 0;
```

bo tutaj widać jak na dłoni po wartości domyślnej że to `:number`

Podobną zasadę stosuj do innych typów prostych.